agent/api/statechange.go

```diff
@@ -153,7 +158,7 @@ func NewContainerStateChangeEvent(task *apitask.Task, cont *apicontainer.Contain
 	return event, nil
 }
 
-func newUncheckedContainerStateChangeEvent(task *apitask.Task, cont *apicontainer.Container, reason string, eventType statechange.EventType) (ContainerStateChange, error) {
+func newUncheckedContainerStateChangeEvent(task *apitask.Task, cont *apicontainer.Container, reason string) (ContainerStateChange, error) {
```

Comment:
I created this function because it was reused by container state change and managed state change, but I believe this is invoked in a single place now? If so, it might be better to just move it to container state change creation funciton.

Reply:
I'll add this to a later PR.